tools/squidclient/squidclient.cc

```diff
@@ -177,6 +178,56 @@ shellUnescape(char *buf)
     *d = '\0';
 }
 
+/// [Proxy-]Authorization header producer
+class Authorization
```

Comment:
Per our style guide this class should be in .h/.cc files of its own. Not a blocker, but it would be nice not to have to come back and shuffle that around later.

Reply:
This helper class is not meant to be reusable elsewhere (without lots of additional work). No (valid) guidelines can require its isolation.